Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8
+
+WORKDIR /app
+
+COPY ./src .
```

Comment:
The `COPY` command has an extra dot (`.`) which might cause confusion. It should be `COPY ./src /app` to copy the contents of the `src` directory into the `/app` directory in the container.